### DISCUSSION TITLE
add`asset_prefix` to assets

### DISCUSSION THREAD

**Discussion by scottyhq** [developmentseed/cogeo-mosaic#226] [Q&A]
I tired rendering the 0.0.3 example which uses public data:
https://titiler.xyz/mosaicjson/map?url=https://raw.githubusercontent.com/developmentseed/mosaicjson-spec/main/0.0.3/example/dg_post_idai.json 

But get a lot of 500 errors, I also tried modifying the docs example to just use `"asset_prefix":"s3://noaa-eri-pds/2020_Nashville_Tornado/20200307a_RGB/"`:
https://titiler.xyz/mosaicjson/map?url=https://gist.githubusercontent.com/scottyhq/4c288a2dbf7b97ef7833c5e0ee19d6d6/raw/636083cf7117a96e38d271eeef932a3ee507d7d1/NOAA_Nashville_Tornado_v3.json

So I suspect titiler by default doesn't have logic to pick up the `asset_prefix`

**Comment by vincentsarago** [comment]
> So I suspect titiler by default doesn't have logic to pick up the asset_prefix

Well I can't remember why but cogeo-mosaic backends themself do not have the logic to add the prefix 😓 


it should be added somewhere here https://github.com/developmentseed/cogeo-mosaic/blob/main/cogeo_mosaic/backends/base.py#L232-L245
